lib/smart_proxy_container_gateway/container_gateway_main.rb

```diff
@@ -100,10 +100,7 @@ def insert_token(username, token, expire_at_string, clear_expired_tokens: true)
       end
 
       def initialize_db
-        conn = Sequel.postgres(host: Proxy::ContainerGateway::Plugin.settings.postgres_db_hostname,
-                               user: Proxy::ContainerGateway::Plugin.settings.postgres_db_username,
-                               password: Proxy::ContainerGateway::Plugin.settings.postgres_db_password,
-                               database: Proxy::ContainerGateway::Plugin.settings.postgres_db_name)
+        conn = Sequel.connect('sqlite://smart_proxy_container_gateway.db')
```

Comment:
Should this be configurable?

Reply:
do you know what directory this uses by default?

Reply:
if it uses the home directory (/usr/share/foreman-tasks), it might be better to set it to /var/lib/foreman-tasks/ IMO.  I think we probably should make it configurable, but just let the default go through

Reply:
Looks like it would stick the database in the source location for foreman-proxy, so yeah /usr/share/foreman-proxy.  I'll make the default /var/lib/foreman-proxy and make it configurable.

Reply:
Alright, this should be all set.